src/utils/structuredData.ts

```diff
@@ -89,3 +88,21 @@ export const structuredOrganization = ({
     'https://www.linkedin.com/company/hedvig/',
   ],
 })
+
+export const structuredOrganization = ({
+  description = DEFAULT_ORG_DESCRIPTION,
+}: StructuredOrganizationParams = {}) => ({
+  '@context': 'http://schema.org',
+  ...structuredDataOrganization({ description }),
+})
+
+export const structuredDataReviewSnippet = ({
+  description = DEFAULT_ORG_DESCRIPTION,
+}: StructuredOrganizationParams = {}) => ({
+  '@context': 'http://schema.org',
+  '@type': 'AggregateRating',
+  itemReviewed: structuredDataOrganization({ description }),
+  ratingValue: '4.8',
```

Comment:
Since we have a hardcoded value of our Trustpilot score on the site today I think it's fine to have it here as well for the time being :)